src/puppetlabs/puppetdb/queue.clj

```diff
@@ -0,0 +1,199 @@
+(ns puppetlabs.puppetdb.queue
```

Comment:
Further, it's where all the commands "in flight" live, i.e. for every command that PDB knows about there'll be a cmdref somewhere in here, backed by a stockpile entry referring to a file on disk, which then allows pdb to have a "global" view of what's pending.